spec/acceptance/trusted_external_command_spec.rb

```diff
@@ -14,48 +14,24 @@
   end
 
   context 'apply the class' do
-    it 'applies the class without error' do
-      idempotent_apply_site_pp(Manifests::DEFAULT)
-      expect(mockserver.mock_called?(default_endpoint, default_query_params, 20, 40)).to be true
-    end
-
-    it 'applies the correct yaml settings' do
```

Comment:
@RandomNoun7 I think we should test "creates correct config file" in the unit tests. We can do that by checking that the `trusted_external_command` resource creates the relevant `File` resource with the right YAML content.

Reply:
Testing in unit tests is fine, but it should also be tested in acceptance that the file is created and the content is as expected.

Reply:
Why? Aren't we already testing that when checking that the script works? If the config file wasn't created w/ the right content, how can the script work in the first place? That's what I meant by implementation detail.

Reply:
If you feel strongly that it's not needed I'm fine with that. I'm usually of the attitude that more tests is always better and testing the data storage layer is usually a good idea.